frontend/src/component/common/Truncator/Truncator.tsx

```diff
@@ -59,6 +60,10 @@ export const Truncator = ({
         return () => resizeObserver.disconnect();
     }, [title, children]);
 
+    useEffect(() => {
+        onSetTruncated?.(isTruncated);
```

Comment:
Is that optional chaining gonna work? I thought you had to do `onSetTruncated && onSetTruncated(args)`? Also, what exactly do you need this one for? There's a lot of things being passed everywhere; I don't remember exactly what it was now

Reply:
That's for "should this list be expandable"